pages/contribute.md

```diff
@@ -6,44 +6,58 @@ main_nav_id: contribute
 
 # Contributing to Kotlin
 
-Kotlin is an Apache 2 OSS Project. The source code, tooling, documentation and even this web site is maintained on [GitHub](https://github.com/jetbrains/kotlin). While the majority 
-of the team works at JetBrains, there have been nearly a hundred external contributors to the Kotlin Project and we are always looking for more people. 
+Kotlin is an Apache 2 OSS Project. The source code, tooling, documentation and even this web site is maintained on [GitHub](https://github.com/jetbrains/kotlin). 
+While Kotlin is mostly developed by JetBrains, there are hundreds of external contributors to the Kotlin project, and we always look for more people&mdash;[become one of them](https://kotlinlang.org/community). 
```

Comment:
and we are always on the lookout for more people to help us
and we are always welcome to having new people 
and we always like to have new people

but we always welcome more

or cut this bit entirely to:
While Kotlin is mostly developed by JetBrains, there are hundreds of external contributors to the Kotlin project [become one of them](https://kotlinlang.org/community).

Reply:
Decided to shorten. Thx for options!